pkgs/racket-doc/scribblings/reference/stx-trans.scrbl

```diff
@@ -923,11 +923,48 @@ definition context.
 
 @defproc[(syntax-local-introduce [stx syntax?]) syntax?]{
 
-Produces a syntax object that is like @racket[stx], except that the
-presence of @tech{scopes} for the current expansion---both the
-macro-introduction scope and the use-site scope, if any---is flipped
-on all parts of the syntax object. See @secref["transformer-model"] for information
-on macro-introduction and use-site @tech{scopes}.
+Produces a syntax object that is like @racket[stx], but with a new 
+@tech{scope} added that flips the apparent introduction site for all 
```

Comment:
I think it's inaccurate to say that it's a "new scope". What's flipped are two existing scopes.